### DISCUSSION TITLE
How to enable remote access without user validation

### DISCUSSION THREAD

**Discussion by dbeniamine** [rustdesk/rustdesk#4507] [Q&A]
Hello,

First of all rustdesk is awesome, thanks for your amazing work !

I have a usage where I want to deploy rustdesk on many computers and have several technicians that are always allowed to access the other computers.

I saw the whitelist IP feature that can be used in my use case to make sure that only the IPs of the technician can access the clients.

But I am looking for a way to avoid the password authentication.

Using the same randomly generated password for all clients would be an acceptable solution, but from what I see in the configuration file, this password is encrypted.

So is there a way to either : 

+ Tell rustdesk that the IP whitelist is the only security
+ Write a script that generates rustdesk configuration for all clients with the same password and whitelist IPs

Best regards

**Comment by SergeyMy** [comment]
the password is encrypted with a **unique client ID** UID

**Reply by dbeniamine** [reply]
yeah that what I though, but the question is, is there (or will there be) a way to configure easily (via CLI) many desktops to allow a restricted number of person (restriction by IP seems enough for me) to have an easy access without human intervention on the controlled machine ?

So either a list of IP for which we can connect without password or a way to set the password via CLI ?

**Reply by SergeyMy** [reply]
the password can be changed / set from the console, as well as look at the ID

Ps setting passwords via config or externally - not secure
to set a password - you must have access to the system! and i agree with that

**Reply by dbeniamine** [reply]
I did not found the CLI documentation, and `rustdesk --help` only starts the client, do you have an example on how I can do that ?

**Reply by SergeyMy** [reply]
https://github.com/rustdesk/rustdesk/discussions/4646
